scripts/fuzz_opt.py

```diff
@@ -329,6 +329,9 @@ def __init__(self):
                     wabt_bin = shared.which('wasm2c')
                     wabt_root = os.path.dirname(os.path.dirname(wabt_bin))
                     self.wasm2c_dir = os.path.join(wabt_root, 'wasm2c')
+                    if not os.path.isdir(self.wasm2c_dir):
+                        print('wabt found, but not wasm2c support dir')
+                        self.wasm2c_dir = None
```

Comment:
I don't see any handling later of the case where `wasm2c_dir` is None. Will it silently skip this check or crash?

Reply:
Line 340 has this:
```python
            def can_run(self):
                if self.wasm2c_dir is None:
                    return False
```

Reply:
👍 Excellent, thank you